ape_safe.py

```diff
@@ -34,15 +34,15 @@
     1: 'https://safe-transaction.mainnet.gnosis.io',
```

Comment:
```suggestion
    1: 'https://safe-transaction-mainnet.safe.global',
```
All networks updates have completed as per the schedule [here](https://forum.gnosis-safe.io/t/transaction-service-migration-october-2022/1550)